notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb

```diff
@@ -0,0 +1,418 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        kwargs["raw_audio_path"] = src_audio
</pre><p>Can other values be parametrized?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Some adjustable params were added